### PR TITLE
Disable blinking airfields for non-remaster builds. Blinking airfields are a change which Petroglpyh added to remaster

### PR DESCRIPTION
**Pull request by mvdhout1992**
Disable blinking airfields for non-remaster builds. Blinking airfields are a change which Petroglpyh added to remaster

### CODE REVIEW

File: redalert/bdata.cpp [hunk 1]
```diff
@@ -2998,8 +2998,12 @@ void BuildingTypeClass::One_Time(void)
                   {STRUCT_FAKEWEAP, BSTATE_ACTIVE, 0, 1, 0},
                   {STRUCT_FAKEWEAP, BSTATE_IDLE, 0, 1, 0},
                   {STRUCT_IRON_CURTAIN, BSTATE_ACTIVE, 0, 11, 3},
+#ifdef REMASTER_BUILD
                   {STRUCT_TESLA, BSTATE_ACTIVE, 0, 10, 2},
                   {STRUCT_AIRSTRIP, BSTATE_IDLE, 0, 8, 3}};
+#else
+                  {STRUCT_TESLA, BSTATE_ACTIVE, 0, 10, 2}};
```
Comment (hifi) [inline redalert/bdata.cpp]: Better to just move the closing brace on its own line and ignore that the formatting is a bit wonky so you don't need to copy the STRUCT_TESLA line over.

**Review by hifi — CHANGES_REQUESTED** [review]

File: redalert/bdata.cpp (hunk 1 above)
Comment (mvdhout1992) [reply redalert/bdata.cpp]: Will update and do just that. Yeah I initially wanted to do the brace on a newline but went with this cause it looks less wonky. Will do it on a newline instead.

**Comment by tomsons26** [comment]
Yeah afaik this is the only change, that entry isn't in 3.03.

**Review by tomsons26 — APPROVED** [review]

**Comment by hifi** [comment]
Just clean up the ifdef block so you don't need to repeat the closing `};` and I'll merge this.

**Review by hifi — APPROVED** [review]

**Pull request merged**